modules/ensembl/metadata/genomesummary/tests/main.nf.test.snap

```diff
@@ -30,56 +30,47 @@
             "nf-test": "0.9.2",
             "nextflow": "24.10.3"
         },
-        "timestamp": "2025-03-25T16:42:09.027167253"
+        "timestamp": "2025-06-18T11:41:51.128599882"
     },
     "Metadata 'genome summary' test-A [real test]": {
         "content": [
-            {
-                "0": [
-                    [
-                        {
-                            "accession": "GCA_000001215.4"
-                        },
-                        "genome.json:md5,75d8680e4d2ca9f53cc3c9939d4d0aa2"
-                    ]
-                ],
-                "1": [
-                    "versions.yml:md5,f2b370b16189c705b60651d56a20c96c"
-                ],
-                "meta_json": [
-                    [
-                        {
-                            "accession": "GCA_000001215.4"
-                        },
-                        "genome.json:md5,75d8680e4d2ca9f53cc3c9939d4d0aa2"
-                    ]
-                ],
-                "versions": [
-                    "versions.yml:md5,f2b370b16189c705b60651d56a20c96c"
-                ]
-            }
+            [
+                "versions.yml:md5,f2b370b16189c705b60651d56a20c96c"
+            ]
         ],
         "meta": {
             "nf-test": "0.9.2",
             "nextflow": "24.10.3"
         },
-        "timestamp": "2025-06-17T11:29:10.474150568"
+        "timestamp": "2025-06-18T11:41:40.716467645"
     },
     "genome.json": {
         "content": [
             [
-                [
-                    {
-                        "accession": "GCA_000001215.4"
-                    },
-                    "genome.json"
-                ]
+                "{",
+                "    \"annotation\": {",
+                "        \"provider_name\": \"GenBank\",",
+                "        \"provider_url\": \"https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_000001215.4\"",
+                "    },",
+                "    \"assembly\": {",
+                "        \"accession\": \"GCA_000001215.4\",",
+                "        \"provider_name\": \"GenBank\",",
+                "        \"provider_url\": \"https://www.ncbi.nlm.nih.gov/datasets/genome/GCA_000001215.4\",",
+                "        \"version\": 4",
+                "    },",
+                "    \"genebuild\": {",
+                "    },",
+                "    \"species\": {",
+                "        \"scientific_name\": \"Drosophila melanogaster\",",
+                "        \"taxonomy_id\": 7227",
+                "    }",
+                "}"
```

Comment:
Is there an option to avoid hard-coding here a test file content? It makes it slightly more complex to maintain.

Reply:
its the output from snapshot